ipaclient/plugins/dns.py

```diff
@@ -62,7 +62,7 @@ def __get_part_param(rrtype, cmd, part, output_kw, default=None):
 def prompt_parts(rrtype, cmd, mod_dnsvalue=None):
     mod_parts = None
     if mod_dnsvalue is not None:
-        name = record_name_format % rrtype.lower()
+        name = record_name_format % rrtype.lower().decode("utf-8")
```

Comment:
This will unfortunately throw an attribute error:
`AttributeError: 'str' object has no attribute 'decode'`

Reply:
@stlaz mentioned he has already seen similar issue.

Reply:
@Tiboris 4.5 is python 2.7, right? So it will not throw any attribute error.

Reply:
@slaykovsky That is true! Sorry.
Have you tried to reproduce this bug on master?

Reply:
@Tiboris on master it's okay :)

Reply:
Should not we be at least `.decode('utf-8')` entire `name` and not 'only' `rrtype`?

And one not very relevant but to stay consistent could you please use 'utf-8' used everywhere in code so far, instead of "utf-8" ? 😉

Reply:
> Should not we be at least .decode('utf-8') entire name and not 'only' rrtype?
After formatting, `name` will become utf-8 string, e.g.:
```
>>> a = "a %s"
>>> type(a)
<type 'str'>
>>> a % "b".decode("utf-8")
u'a b'
```

> use 'utf-8' instead of "utf-8"
Sure :)

Reply:
"4.5 is python 2.7" is a not-good-enough reasoning not to fix this properly (meaning compatible with python3 - such incompatibilities often show poor design choices).